Sources/StytchCore/ClientInfo/ClientInfo+Version.swift

```diff
@@ -1,3 +1,3 @@
 extension Version {
-    static let current: Self = .init(major: 0, minor: 26, patch: 0)
+    static let current: Self = .init(major: 0, minor: 25, patch: 1)
```

Comment:
is the minor patch supposed to still be `26`?

Reply:
No, this is one of the weird things with the way the iOS app does versioning. The latest packaged release is 0.25.0, but the code in main is already set to be 0.26.0 (this happens automatically when you create a release through the github actions). This is a patch to the last released version, so it should be 0.25.1.

This is also the first time I've had to do a patch release for iOS, so I'm thinking a lot of the tooling isn't going to work 😩

Reply:
Doing my once-every-few-months drop in on the repo so figured I'd chime in: The intent for patch releases with the way things were set up initially was that patch releases get their own branch created, fixes get cherry picked into that branch, and then a tag is made from that branch once it's ready.